src/main/java/com/vmware/vipclient/i18n/messages/service/ComponentService.java

```diff
@@ -31,17 +33,24 @@ public ComponentService(MessagesDTO dto) {
     }
 
     /*
-     * get messages from local bundle or from remote vip service(non-Javadoc)
+     * Get messages from local bundle or from remote vip service(non-Javadoc)
      * 
      * @see
      * com.vmware.vipclient.i18n.messages.service.IComponentService#getMessages
      * (com.vmware.vipclient.i18n.base.DataSourceEnum)
      */
     @SuppressWarnings("unchecked")
-    public Map<String, String> getMessages() {
+    public Map<String, String> getMessages(final Map<String, Object> cacheProps) {
         Map<String, String> transMap = new HashMap<String, String>();
+        ComponentBasedOpt cbo = new ComponentBasedOpt(dto);
```

Comment:
Do you think opt is more clear?

Reply:
What does "opt" mean in ComponentBasedOpt?

Reply:
Anyway, I changed it to opt.